staging/src/k8s.io/client-go/discovery/cached/disk/cached_discovery_test.go

```diff
@@ -36,72 +36,72 @@ import (
 )
 
 func TestCachedDiscoveryClient_Fresh(t *testing.T) {
-	assert := assert.New(t)
+	assrt := assert.New(t)
```

Comment:
nit: Why this pr renames here?

Reply:
It may not be the suggested way to use package name as variable name, thus I changed it. It has no effect on tests.

Reply:
Having the variable name different from the package name is better, but I'm afraid that we will get PRs in the future thinking `assrt` was a typo and reverting the change. How about we name it `as` (used in some files in the code base) or `assertions` (what [New returns](https://pkg.go.dev/github.com/stretchr/testify/assert#New))?

Reply:
Thanks @roycaihw for the suggestion, `as` is a good idea and changed that way.